[EDU-5478] refactor: update existing docs to add information related to botManagerBreakdownMetrics dataset and graphs (#1322)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx

This information helps you identify regional patterns and implement geo-blocking and other region-specific mitigation strategies.

#### Breakdown 

##### Impacted URLs

> The Impacted URLs graph shows the total number of requests detected as bots, calculated based on the number of URLs that experienced bot activity.

Author: ```suggestion
The Impacted URLs graph shows the total number of distinct URLs bots request.
```
